### ISSUE TITLE
[misc] Custom cron.d crontabs don't necessarily generate emails

### ISSUE THREAD

**Issue by Alphix** [debops/debops#2162]
As noted in issue #2147, roles which create custom ``/etc/cron.d/<foo>`` crontabs with non-root users might end up not generating any emails on failure at all (since the user might not have a suitable alias).

This isn't limited to `rspamd` though...there's other roles which seem to do something similar, e.g.:

* roundcube
* owncloud
* docker_registry
* postwhite

Not sure what the best approach is. Should we simply ignore this or should all the roles be fixed up (e.g. by setting ``MAILTO=`` in the relevant crontabs, or by adding appropriate entries to ``/etc/aliases``)?

@drybjed : opinions?

**Comment by drybjed** [comment]
It all depends on if the local mail is enabled. With `nullmailer` being the default SMTP service, local mail is not possible and all mail not to a specific known user should be sent to `root` account configured in `/etc/nullmailer/adminaddr`. Since all mail is sent to the mail relay, it's that relay's responsibility to deliver it to apropriate admin accounts.

When the `postfix` role is used, local mail is enabled by default. The role includes the `etc_aliases` role in the playbook, so administrators can configure apropriate e-mail aliases for local mail there. But it might get tricky for system accounts like `_rspamd`, with the defaults mail to these will end up in the local mail spool directory.

I guess we could go with a `MAILTO=` variable in each crontab entry that's based by default on `ansible_local.core.admin_public_email`. That way system accounts could deliver their mail to sensible admin e-mail addresses by default. I'm iffy about the variable management by the `cron` Ansible module, it's weird... I suppose if we generate the `cron` entries ourselves that's fine. Each role should have `<role>__admin_mail_to` variable for this purpose, that could cover `cron` and other cases for such e-mails like account registration, etc.

**Comment by Alphix** [comment]
> I guess we could go with a `MAILTO=` variable in each crontab entry that's based by default on `ansible_local.core.admin_public_email`. That way system accounts could deliver their mail to sensible admin e-mail addresses by default. I'm iffy about the variable management by the `cron` Ansible module, it's weird...

Yeah, looking at [this](https://stackoverflow.com/questions/64285237/ansbile-way-of-having-multiple-mailto-environment-varialbes-in-crontab), the `cron` module seems pretty wonky.....I guess one option would be to ignore the `cron` module and use templates instead...

**Comment by mathieumd** [comment]
I think @drybjed suggestion to use `ansible_local.core.admin_public_email` in `MAILTO` would work, as it's supposed to be a single mail address. Even with several jobs with each their own`MAILTO` in the same crontab file, they would all still have the same value.
